### ISSUE TITLE
Bug: listMonth shows weekends even if "weekends: false"

### ISSUE THREAD

**Issue by deap82** [fullcalendar/fullcalendar#3644]
Recreation: http://jsbin.com/taxogufaqa/edit?js,output

Docs for weekends options says "columns", but also "any of the calendar views";
https://fullcalendar.io/docs/display/weekends/

What's the expected behaviour? Is there another way to hide weekends in the list views?

**Comment by arshaw** [comment]
You're right, `weekends:false` should probably days with events in them on listview.

As a workaround, you might want to try [eventRender](https://fullcalendar.io/docs/event_rendering/eventRender/) and return `false` for events that fall on weekends. If there are no events on a given day, the day won't display at all in listview.
